examples/azure-rules.yaml

```diff
@@ -656,4 +656,51 @@ groups:
           Please, check the naming convention in case the resource configuration is done correctly.
            
           More information at https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability#create-sap-hana-cluster-resources
+        scored: true
+  - id: 1.6
+    description: "Check OS and package versions are supported and up-to-date"
+    checks:
+      - id: 1.6.1
+        description: "Check os release"
+        audit: 'cat /etc/os-release'
```

Comment:
how does that work? does this then only match this in *any* line or in the first line?

can we do something lik e

`. /etc/os-release ; echo $NAME-$VERSION_ID`

Reply:
It didn't work for me when I did `source /etc/release && echo $PRETTY_NAME`, so I did it in the uglier way. There is an `bin_op: or`, so it checks if any of the flags is in the /etc/os-release.